### ISSUE TITLE
Draw:Polyline delete vertex 

### ISSUE THREAD

**Issue by frulo** [Leaflet/Leaflet.draw#523]
In the Edit.Polyline modus I can delete any vertex with a click. It this also possible in the draw.polyline modus?
I know that I can delete the last vertex, but I want to allow the user to delete also the other vertices.

**Comment by ddproxy** [comment]
Yes, you can use the edit function of a line (after it's been completed) to click a vertex and delete it.

**Comment by frulo** [comment]
Yes, I know this function, but I want to be able to delete a vertex before the user complete the Polygon.
I already wrote an extension that allows the user to delete a vertex with ctr+click (to avoid conflicts with other mouse interactions). Actually I think that this is a very special solution for our application, but if there is an interest to this functionality I can make a pull request.
